collectors/freebsd.plugin/multi_metadata.yaml

```diff
@@ -1111,14 +1185,19 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
-          description: ""
+          name: "netdata.conf"
+          section_name: "plugin:freebsd"
```

Comment:
Please add the brackets to the rest, from checking diskspace.plugin and how it renders, I understand we need brackets in these